server/log_operation_manager.go

```diff
@@ -224,7 +224,14 @@ func (l *LogOperationManager) masterFor(ctx context.Context, allIDs []int64) ([]
 
 	held := l.tracker.Held()
 	heldIDs := make([]int64, 0, len(allIDs))
+	sort.Strings(allStringIDs)
 	for _, s := range held {
```

Comment:
Looks like the "active as master for N/M" message below compares chickens to tables.

Reply:
Haha I was going to mention that, but since this change doesn't actually resign mastership of the frozen/deleted trees, it seemed a bit out of scope. Come to think of it though, `len(allIDs)` will decrease when a tree is frozen/deleted, but `len(held)` won't, will it? That would probably be confusing.

Reply:
Dome some logging/commenting rearrangement. PTAL.